Makefile

```diff
@@ -1,9 +1,11 @@
 # Some simple testing tasks (sorry, UNIX only).
 
+pytest := python3 -m pytest
+
 all: test
 
 .install-deps: requirements/dev.txt
-	@pip install -U -r requirements/dev.txt
+	@pip3 install -U -r requirements/dev.txt
```

Comment:
aiohttp development assumes virtual environment usage.
In proper virtual env `pip` is just an alias for `pip3` if python is python3